easy-cluster/easy-cluster.sh

```diff
@@ -716,13 +728,17 @@ function run_job() {
     az batchai job create -n $JOB_NAME --cluster-name $CLUSTER_NAME -c job.json -g $RG -l $LOC -o table
     echo
     echo -e "${GRAY}  You can see the progress of your job in the portal now or run the following command"
-    echo -e "  az batchai job show -n sample-job-15935 -g $RG -o table"
+    echo -e "${GREEN} https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/61fcfcb0-5cbf-4081-94a3-4be06a6e153d/resourceGroups/${RG}/providers/Microsoft.BatchAI/jobs/${JOB_NAME}/job-overview"
```

Comment:
The `61fcfcb0-5cbf-4081-94a3-4be06a6e153d` is the Ascend Linux sub...would we want this subscription id to be dynamic?

Reply:
No! lets remove that